packages/app/src/specs/SpecsListBanners.vue

```diff
@@ -16,7 +16,7 @@
     <p>{{ t('specPage.noSpecError.explainer') }}</p>
   </Alert>
   <Alert
-    v-if="showOffline"
+    v-else-if="showOffline"
```

Comment:
I know the new banners have mutually exclusive logic, but how about adding a test to confirm the offline or fetch error banner take precedence and prevent the new banners from showing?

Reply:
Added tests to `SpecsListBanners.cy.tsx` to validate that each smart notification is preempted by each of the 5 higher-priority banners [`da3c2e3`](https://github.com/cypress-io/cypress/pull/23256/commits/da3c2e3be43d84ccb4816bc2a08b341c394462d3)

Reply:
good tests!